cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/StopStartUpscaleStartInstancesHandler.java

```diff
@@ -165,7 +165,8 @@ private List<CloudVmInstanceStatus> getInstanceStatusOnStartError(CloudConnector
     private List<CloudVmInstanceStatus> collectStoppedInstancesFromCloudProvider(
             CloudConnector connector, AuthenticatedContext ac, List<CloudInstance> cloudInstances) {
         List<CloudVmInstanceStatus> vmInstanceStatusAllInstances = connector.instances().checkWithoutRetry(ac, cloudInstances);
-        return vmInstanceStatusAllInstances.stream().filter(vm -> vm.getStatus() == InstanceStatus.STOPPED).collect(Collectors.toList());
+        return vmInstanceStatusAllInstances.stream().filter(vm -> (vm.getStatus() == InstanceStatus.STOPPED) || (vm.getStatus() == InstanceStatus.PENDING))
+                .collect(Collectors.toList());
```

Comment:
If we're trying to recover during an UPSCALE - should we be doing the same during a DOWNSCALE as well ?